src/count-blog-domain.js

```diff
@@ -7,9 +7,9 @@
 const url = require('url');
 
 //  import logger
-const parentLogger = require('./lib/logger');
+const { logger } = require('./lib/logger');
 
-const log = parentLogger.child({ module: 'count-blog-domain' });
+const log = logger.child({ module: 'count-blog-domain' });
```

Comment:
One thing we'll have to do in another PR is get better at naming our "modules" for the logger.  Each class doesn't need its own module name.  Rather, we should lump them together into logical components in the app.